pulsar-core/src/pdk/process_tracker.rs

```diff
@@ -204,6 +208,7 @@ impl ProcessTracker {
                         exit_time: None,
                         original_image: self.get_image(ppid, timestamp),
                         exec_changes: BTreeMap::new(),
+                        argv: Vec::new(),
```

Comment:
forked process probably should have the same argv of the parent

Reply:
Good catch